### DISCUSSION TITLE
How to fix this typescript error

### DISCUSSION THREAD

**Discussion by devantic** [nativewind/nativewind#952] [General]
![image](https://github.com/nativewind/nativewind/assets/32611040/b7df7f03-00b4-41a1-8cef-9dc963afd9f5)

I've spend way too long trying to remove this one squiggly red line :(

**Comment by MikeChongCan** [comment]
likely the react native / react typing version does not match

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
